packages/dashboard/src/components/resourceExplorer/describeCurrentAsset.ts

```diff
@@ -0,0 +1,41 @@
+import { IoTSiteWiseClient, DescribeAssetCommand, DescribeAssetResponse } from '@aws-sdk/client-iotsitewise';
+import { HIERARCHY_ROOT_ID } from '.';
+
+const blank = {
+  assetId: undefined,
+  assetArn: undefined,
+  assetName: undefined,
+  assetModelId: undefined,
+  assetProperties: [],
+  assetHierarchies: [],
+  assetCompositeModels: [],
+  assetCreationDate: undefined,
+  assetLastUpdateDate: undefined,
+  assetStatus: undefined,
+  assetDescription: undefined,
+} as DescribeAssetResponse;
+
+export const sendCommand = (client: IoTSiteWiseClient, assetId: string) =>
+  client.send(new DescribeAssetCommand({ assetId }));
+
+export const describeCurrentAsset = async (currentBranchId: string, client: IoTSiteWiseClient | undefined) => {
+  if (!client) return blank;
```

Comment:
I thought this was some new return type for undefined vs null. I was confused for a second 😄